content/intuitionistic-logic/soundness-completeness/soundness-nd.tex

```diff
@@ -10,6 +10,20 @@
 
 \olsection{Soundness of Natural Deduction}
 
+We will now prove soundness of natural deduction with regards to the 
+relational semantics, that is, showing that if a formula is deducible from
+a set of assumptions then the set of assumptions entails the formula. Observe that
+when proving that $\Gamma \Entails !A$ we may use the definition
+\begin{itemize}
+  \item for every model~$\mModel{M}$ and every~$w$ such that
+$\mSat{M}{\Gamma}[w]$, $\mSat{M}{!A}[w]$
+\end{itemize}
+or the result of \olref[int][sem][sem]{prop:sat-entails} and prove
+\begin{itemize}
+  \item if for every model~$\mModel{M}$ such
```

Comment:
Nice catch that this is what the proof actually uses. It would be less confusing & cleaner though if we fixed the proof instead.  After all, we don't prove anywhere that (say) if M |= A and M |= B then M |= A /\ B. But M, w |= A and M, w |= B implies M, w |= A /\ B by definition.

Reply:
@feffemannen what do you think? Let me know if you want to make that change and add it to the PR, or if you'd like me to do it after.

Reply:
Well, yes. But you would get a lot more ",w" appearing the proof, without any real use. Either way would be fine with me. Please make the changes you think are most appropriate.